ec/fake_ec.go

```diff
@@ -54,6 +55,14 @@ func (ts *Tipset) Timestamp() time.Time {
 	return ts.timestamp
 }
 
+func (ts *Tipset) String() string {
+	res := base64.RawStdEncoding.EncodeToString(ts.tsk[:gpbft.CidMaxLen])
```

Comment:
I'd check the length before slicing.

Reply:
Honestly, if we're going to stringify the tipset, I'd prefer to just stringify it as proper CIDs.

Reply:
If the length is wrong, then it means someone messed with it. The FakeEC will always create proper length, the test is there so if anyone changes it, it gets detected.

I'm importing multibase, at this point we might import go-cid but I'm not touching that right now.